
# The Human Pangenome Project: Creating A Global Community-Driven Resource

A human pangenome reference represents an advanced genomic coordinate system that offers a better representation of global genome diversity. If built well and adopted broadly, it will serve as a critical foundation for more equitable genomic research. This resource aims to remove inherent biases and broaden studies of genetic variants that impact biomedical research and precision medicine. A project of this magnitude and impact on human health requires broad global participation. The legacy of this project will be defined not only by its technical achievements, but also by the number of researchers, clinicians, and patients around the world who benefit from its creation and use. Here we provide an initial framework for the Human Pangenome Project (HPP), a global initiative to construct and adopt a reference resource to serve as the bedrock for a new era of precision medicine and basic research. This white paper is a call to action for the global community to work together to develop a human pangenome resource. 

## Mission Details 
The human reference genome is a centralized coordinate system that is used by researchers as a standard for comparing every human DNA sequence. It was first produced by the International Human Genome Project over twenty years ago 1,2, and has been updated on a regular basis since by the Genome Reference Consortium 3. It has served as the essential resource for biomedical research, enabling important breakthroughs that have benefited science and medicine. Its role in enabling research to identify genetic links to disease susceptibility and drug response has become integral to precision medicine4. However, the current human reference genome is stitched together using pieces of genomes from a small number of individuals, where every genomic location contains a sequence from a single individual. It is also incomplete, missing at least 8% of the genome, including more than 100 genes, and contains a number of known errors 5. This limits the accuracy to identify differences across genomes (or genetic variants), which are necessary to better understand the impact of human genetic diversity on our biology. A reference that better represents the rich landscape of human genetic diversity will be an important resource for uncovering and evaluating a much greater complexity of variants and increasing the accuracy of genomic variant discovery and interpretation, even in genetically understudied groups. A new type of reference genome - called a human pangenome reference - that is more accurate, more complete, and portrays human genomic diversity at a broader scale is needed. It must simultaneously represent more diverse genetic variants found globally across our human species, and reduce the errors in the current human reference. Reducing bias will improve our understanding of human biology, enhance disease-association research, advance precision health and therapeutics, and promote equitable healthcare. 

In 2019, a community of scientists dedicated to addressing this need formed the Human Pangenome Reference Consortium (HPRC) with funding from the National Human Genome Research Institute (NHGRI) of the United States (US) National Institutes of Health. The HPRC initiated the construction of a new pangenome reference 6. Unlike previous reference genomes, this pangenome would simultaneously represent hundreds, and could eventually include thousands, of complete maps of many diverse genomes. To generate these maps, we have developed new community standards in end-to-end (also called telomere-to-telomere) completion of human genomes that are dramatically expanding studies of complex and hard-to-map variants. For the first time in a human reference, the chromosomes inherited by the biological father and mother are represented separately, helping to pinpoint inherited variants leading to more accurate diagnoses. Already, some biomedically relevant complex variants are starting to be resolved that address previously intractable biomedical problems. For example, the human leukocyte antigen (HLA) region, with its key roles in the immune system, is now fully assembled 5,7. This reveals the diversity of HLA gene variants, with implications for matching patients and donors for bone marrow or cord blood transplant, as well as an improved understanding of the workings of the human immune system. Another example is the survival motor neuron genes (SMN1 and SMN2), which are difficult to sequence and interpret and yet are important targets for gene therapy associated with spinal muscular atrophy 8. As the pangenome is increasingly adopted, these more complete maps containing many new genes will yield a trove of new biomedical insights.

To understand the pangenome map better we use a train map as an analogy. A pangenome represents a set of individual complete genomes that are combined together into a composite data structure, similar to the graphical representation of the routes and stations in a train map. Shared routes represent shared sequences. Imagine a city with a large rail system with many lines. Some lines, or routes, are used by almost everyone who takes the train. These are like core sequences, shared by most individuals within our species. Then, some routes diverge at specific locations, just like sequence variation adds diversity to different individuals. So, the pangenome is like the entire map of this rail system, encompassing all the lines - and if truly representative of our species, every person on the planet will be able to trace a route most similar to their genome through this map from start to finish. In 2023, several groups provided initial pangenome drafts 9–12. For example, the HPRC published the first globally diverse human pangenome draft 12, featuring high-quality assemblies of 47 genomes. Initial maps like these, like rail systems under construction, serve as a foundation to develop a suite of downstream tools, pipelines, and applications. Although in their early stages, these pangenomes are already helping scientists study complex regions of the genome 46, with discoveries that have implications for understanding infertility and congenital conditions 47. Moreover, their rapid adoption in bioinformatic pipelines demonstrates their potential to significantly increase the accuracy of variant identification within genomes 13,14. However, despite this potential to reduce bias and advance science and medicine, several challenges and barriers exist that could inhibit widespread adoption15. Early input from global communities and experts across many sectors is needed to guide the development of this resource to ensure the benefits can be optimally realized. 

## The Human Pangenome Project
To ensure a human pangenome benefits the global community, it's clear that a united global effort is needed. Accordingly, an international consortium known as the Human Pangenome Project (HPP) has been established as a driver project of the Global Alliance for Genomics and Health (GA4GH) (Figure 1), which is an international nonprofit alliance aimed at accelerating health and research advancements through the ethical, effective sharing of genomic and clinical data. HPP aims to build collaborative relationships among international researchers, academia, industry, government agencies, and participants. In HPP, the HPRC (a US-led initiative) would be positioned as one of many pangenome initiatives worldwide and will engage and work closely with a federation of global partners. Global outreach via HPP will focus on community engagement, outreach programs, knowledge exchange mechanisms, governance requirements, educational programs, and the development of multilingual outreach materials. These activities will help to ensure global participation and local contribution to improve these global resources. Through these collaborative efforts, the HPP will guide the development of an integrated global pangenome resource.

A key aspect of HPP is the value added by including relevant knowledge of ethical, legal, justice, inclusion, and social issues that affect local individuals, communities, and society as a whole. These issues include concerns about privacy, identifiability, commercialization, and intellectual property.  They also include matters of informed consent, data sharing, benefit sharing, risks associated with non-participation, and trust and transparency. Additional political and ethical complexities include emerging issues of ethnic discrimination based on information about genomic diversity, and the development of genomic technologies, like gene editing. These issues require scholarship and expertise tailored both to relevant local contexts and goals of international access and utility, to ensure responsible research and well-developed ethical frameworks. Therefore, in addition to addressing the technical challenges of constructing the highest quality pangenome reference, the success of this project will be defined by ‘who has access to this new technology’, ‘who is benefiting through improved health outcomes’, and ‘how to support a global workforce capable of studying medically relevant variants using the human pangenome’. In short, it is not enough to build a new reference pangenome that is more representative of the diversity of our species -we must also work together to make sure that this new resource is globally accessible to advance the promise of precision medicine around the world. The goal of the HPP is that every human on the planet has an equal opportunity to benefit from the pangenome. This white paper is a call to action for the global community to join forces in creating this new resource.

## Precedents to build on 
The HPP builds on 22 years of genomic research, during which we have learned many lessons about good and harmful practices. These lessons are particularly important with the resource we are developing, which will be publically accessible. This means that the human pangenome can be used for both not-for-profit and for-profit purposes, which can present challenges related to privacy, ethics, commercial exploitation, and security. To meet these challenges, the HPP is building on the work of scholars and organizations throughout the world, including international communities such as the GA4GH 16, the Global Indigenous Data Alliance (GIDA) 17, the International Genetic Discrimination Observatory (GDO) 18,19, Human Heredity & Health in Africa (H3Africa) 20–22, and Australia’s National Center for Indigenous Genomes 23. We are also learning from existing guidelines and policies 24, such as the Canadian Tri-Council Policy Statement 25, Te Mata Ira (Guidelines for Research with Māori) 26,27, Genomic Partnerships - guidelines for genomic research with QIMR Berghofer report on Aboriginal and Torres Strait Islander peoples of Queensland 28, the Native BioData Consortium 29,30, and the National Congress of American Indian's Genetic Resource 31. Researchers from many countries are already developing genomic resources based on Indigenous ethical frameworks 32–39 which provide blueprints for successfully engaging communities. The HPP considers it imperative to align with the principles of Collective benefit, Authority to control, Responsibility, and Ethics (CARE) 40,41 as well as the Findable, Accessible, Interoperable, and Reusable (FAIR) 42 and Transparency, Responsibility, User focus, Sustainability, and Technology (TRUST) principles 43. Aligning the data-centric principles of FAIR, while enshrining the people- and purpose-centric principles of CARE creates opportunities that will require clarity of purpose and innovation. We recognize that not all pangenomes created by different nations and groups will be completely open, acknowledging national laws about exporting samples and data, and respect for sovereignty. By building and developing sustained, meaningful, and reciprocal partnerships with the appropriate groups and organizations, we will face these challenges through thoughtful and innovative approaches to data sharing. We understand that creating an open global resource may bring unforeseen challenges, but we are committed to navigating these while respecting both national and international laws and adhering to ethical principles. Through this white paper, we are inviting global perspectives to help systematically identify and explore issues and to help develop policies, while also drawing on the work others have already done and are continuing to do.

<img width="699" alt="Screenshot 2024-02-21 at 8 40 04 AM" src="https://github.com/ga4gh/human-pangenome-project/assets/44613972/badaf410-dad0-4177-8889-118443d7c23a">

**Figure 1:** The Human Pangenome Project (HPP) is recognized as a Global Alliance for Genomics and Health (GA4GH) Driver Project. In this model, the HPP is an umbrella for many projects in an alliance of global genomics partners united in the goal of constructing a shared global pangenome reference resource, establishing technical standards to ensure resource quality, establishing ethical and policy standards (including pangenomic methods and tools that respect Indigenous data sovereignty and CARE principles), and federated systems for data sharing. This alignment of the HPP with the overarching goal of GA4GH is to create stronger linkages between genomic research and medicine for equitable access to genomics-driven healthcare.


## Sampling and Representation for the Pangenome: Models and Approaches
Recognizing these challenges and opportunities in building an open-access, ethically aligned pangenome resource, the imperative now shifts to understanding the vast spectrum of human genetic diversity. This understanding is to meet both the scientific and the equity goals of this resource, ensuring that the benefits of genomic research are available to all humans. Human genetic diversity refers to the variety of genetic differences found within the human species, as a result of demographic processes such as genetic drift, environmental adaptation, and forced, indentured, and voluntary migration over tens of thousands of years. A fuller understanding of this genetic diversity requires selecting representative samples from many branches of our shared human tree, or lineage, rather than sampling from only a subset of branches. Including more diverse and representative individuals in the pangenome is also an ethical imperative: it is the first step towards ensuring that the benefits of this resource are shared equitably across all humans. Finally, to maximize the impact of this resource, we believe it must be universally accessible, and that education and training for its use must promote inclusion. The question then arises: how can we, as a global scientific community, collaborate to build and refine this resource, ensuring that we establish sampling and representation to have the broadest benefit and best advance science and medicine worldwide?

<img width="726" alt="HPP Fig2" src="https://github.com/ga4gh/human-pangenome-project/assets/44613972/1adf3962-6013-4a22-b4b5-8e52a75eaab4">

**Figure 2:** Three models for selecting samples Two algorithmic approaches, 'MaxVar' focuses on variant-level diversity, 'MaxDiv' assesses genomic divergence, and one approach emphasizes 'Outreach' to include groups currently underrepresented in genomic research. The first two models analyze genetic variants in existing genomic datasets for selection.


Recognizing that multiple well-founded strategies for sampling human genetic diversity exist, we will learn from various approaches that maximize genetic variation, optimize representation across historical human lineages, and foster inclusion and reciprocal benefit-sharing. For example, in the US-led HPRC initiative, three parallel models have been developed for selecting samples (Figure 2). One prioritizes variant-level diversity, a second prioritizes genome-level divergence, and a third prioritizes the inclusion of samples from groups underrepresented in current genomic research. The first two models were implemented as algorithms that select samples by analyzing genetic variants in pre-existing genomic datasets to evaluate the genetic diversity captured by each approach. The first model uses an algorithm to optimally select samples that maximally contribute new genetic variants to the pangenome reference, whereas the second algorithm optimally selects samples that maximally represent human genetic lineages. In theory, these selections can be evaluated in any external genomic dataset to preview how the pangenome would represent variant and genomic diversity, resulting in a flexible way to yield insights into each strategy, and simultaneously compare multiple possible choices of selections. 

Other models for selecting samples, in HPRC and other projects, can focus on individuals and communities currently underrepresented or absent from current genomic resources. For example, it is known that people from regions including Western Asia/Middle East, Eastern Europe, Northern Africa, and Oceania are underrepresented in genomic databases compared to people from North-Western Europe. Likewise, groups of people within some regions of the world may be underrepresented relative to other groups of people in the same regions, for example African-American, Hispanic or Latino(a), and Indigenous American relative to European American in the US. These strategies explicitly prioritize individuals from those groups through processes of partnership, while acknowledging the potential political and ethical sensitivities of this approach. This includes consideration of participants who may have community ties that compel community-based consent protocols and practices that align with cultural values and norms. These partnerships center on the expertise and knowledge of local researchers and include reciprocal training, resource sharing, and legal support to ensure benefit sharing. We anticipate these approaches may capture genomic diversity that is currently overlooked, as there are likely to be variants that are rare or absent in current genomic research databases that have arisen to appreciable frequency in individuals from underrepresented groups. 

By understanding various strategies implemented by HPRC and different pangenome efforts, we plan to learn the risks, benefits, and tradeoffs for capturing human genetic diversity that best facilitates the goal of HPP to build a global reference resource. As more genomic datasets are generated around the world, our understanding of human genetic diversity will grow. Further, the rapid emergence of large-scale, long-read genomic datasets will enable a fuller investigation of larger structural variants in repeat-rich complex regions of the genome, and assess their resolution and frequency in pangenome collections 44. A high priority could be placed on medically relevant genes and variants whose high-resolution representation will improve the utility of the resource for precision medicine. At this time several teams across the world are generating their own human pangenomes, for example, the Chinese Human Pangenome Project 9 and the Arab Human Pangenome Project 10,11.The strategies within each project reflect their unique goals of serving the communities they aim to represent. A challenge for the HPP will be reconciling the common goal across projects of enhancing representation in human genomics while acknowledging each effort may have differences in regional vs global focus, methods for sample selection, scale of sequencing, and data access and use policies. We anticipate that learning will be iterative and it will require evolving our understanding of scientific process and policy.

## Global participation and partnerships 
HPP plans to create a federated alliance of partners worldwide with a shared goal to establish a global pangenome resource. Through collaboration, partners will work together in an inclusive governance structure and a data portal where all global participants will have equitable recognition and representation (see Figure 1). This approach is expected to facilitate ethical and compliant use of genomic data worldwide. To further foster participation and partnerships, the HPP will support the development of local infrastructure and expertise on the use of tools and protocols for generating high-quality reference genomes. Additionally, the HPP will seek to create an educational environment customized to various user groups, offering translated and culturally sensitive resources. Both policy and technical infrastructure need to be developed to establish a cohesive framework for membership and global partnerships that can accommodate myriad data stewardship models and promote reciprocal knowledge exchange. Ultimately, the HPP aspires to provide more equitable access to, and benefit from, the pangenome reference on an international scale.

To further build the partnerships that form the HPP, we propose a two-tiered framework: 1) Core guiding principles promoting values of scientific excellence, equity, transparency, and accountability, and; 2) Specific guidelines for data generation, contributors, and users that respect regional and community governance (see Box 1). Data contributors will be expected to actively involve participants and secure informed consent for the use of samples in the pangenome reference. They should also commit to maintaining data quality and compatibility and following agreed-upon technical standards of HPP members. The HPP will also advocate for responsible data use, support data sharing that strives to uphold local and international regulations, and foster optimum utilization of partnership-created resources. Responsible data use also includes developing strategies to mitigate unauthorized re-identification or misuse, as well as providing guidance on the necessary security standards and regional laws. HPP intends to cultivate a collaborative environment that maximizes the benefits of the pangenome while maintaining the utmost respect and protection for individual contributors and communities. The HPP will also work with members of the GA4GH Regulatory and Ethics Workstream 45, to hone frameworks, policies, and protocols regarding inclusion, data acquisition, and stewardship. Each new partnership will undergo a meticulous evaluation to maintain high scientific, cultural, ethical, and legal standards, fostering relationships grounded in trust and transparency. Educational programs will be implemented including workshops and seminars, both in-person and online, that are geared toward bi-directional sharing of knowledge with members of international genomic data initiatives and the larger scientific community.

<img width="740" alt="HPP Box1" src="https://github.com/ga4gh/human-pangenome-project/assets/44613972/7d3f4d59-d241-4ddb-b02b-9e53fec5ccd7">

## In Partnership, we can Create a more Global, Inclusive, and Beneficial Human Pangenome Reference 
Since the HPP will be developed in partnership with global contributors and serve the international community, it must maintain trustworthiness and scientific excellence. Transparency and trustworthiness cannot be merely adjunct qualities; they must be the foundation upon which the initiative thrives. To actualize this vision, a robust framework, embracing the principles of responsible data sharing, must be instituted. For it to be globally representative, it must be constructed in a manner that is inclusive and respectful – not only of all peoples, but of their communities, and the legal and normative frameworks within which they live. It has to be a resource in which members of many groups put their trust, help foster its development, and commit to sharing benefits equitably. This will be a learning experience for all parties. At the foundation of this effort is resource creation, including the pangenome and tools to make pangenomes useful. The next layer upon this foundation is, and must be, engagement, education, training, and conversation. An initial group of HPP partners has convened (see Box 1), and we anticipate this community growing over time as others decide how and to what extent they will participate. These global partnerships across HPP will build a pangenome resource as best we can, share it as openly as possible, and continue to ask for help to make it globally useful, locally considerate, and simply better.

## Prior Resources:
### Links to recordings and notes from workshops to develop the policy paper:
- April 15, 2021 Population Sampling https://wustl.box.com/s/03vtgigk7gfu1vvuqyol6mnkafop9tqx
- April 9, 2021 Mission Details https://wustl.box.com/s/8ttuttcygtpker9tet7ljhpg0hsagjkr
- April 6, 2021 Partnerships https://wustl.box.com/s/38whiphj77dlvo8ejdqe1ot5bi6pz6ph
### Links to international outreach document:
- https://docs.google.com/document/d/1k9w7z4fHCziPtVdYxukQqnmd-Bs5oD_ILr5RX7RYl4A/edit?usp=sharing
### Previous versions of this paper:
- Version 4 (with track changes):
https://docs.google.com/document/d/1YBnAfr7t2Vj-23ucxbwUYwisre0VI-2CX3u_BMR1Ojg/edit?usp=sharing
- GA4GH International Outreach Working Group (frozen 2/9/24):
https://docs.google.com/document/d/1zq9D5Jay5d1k1W66Cp4Gm4YCMHUti750GfpPBTvV9IQ/edit?usp=sharing
- Steering Committee previous version (frozen 12/20/23)
https://docs.google.com/document/d/1vY2tk6_1OM6vvi4MpvCYj-cgPRZSAJlRfWscJFuzHk4/edit?usp=sharing
- PSR and ELSI Commented previous version (frozen 9/30/23):
https://docs.google.com/document/d/17NVvxJ-CIbxNGb5Rjhgg3zebyl-9M2e8O8I-BdvVqBw/edit
- Version 3: https://docs.google.com/document/d/1aVWJ1KsHTyJObS4eA-at_Dc9nmzyCT5wjhqqms_w7pA/edit
- Version 2: https://docs.google.com/document/d/1bjqLRpMBnBNo4ks_Hfwqe0lkmNts-Tc7r033gTPjiTU/edit

## REFERENCES
1.	Venter, J. C. et al. The sequence of the human genome. Science 291, 1304–1351 (2001).
2.	Lander, E. S. et al. Initial sequencing and analysis of the human genome. Nature 409, 860–921 (2001).
3.	The Genome Reference Consortium. https://www.ncbi.nlm.nih.gov/grc.
4.	Ashley, E. A. Towards precision medicine. Nat. Rev. Genet. 17, 507–522 (2016).
5.	Nurk, S. et al. The complete sequence of a human genome. Science 376, 44–53 (2022).
6.	Wang, T. et al. The Human Pangenome Project: a global resource to map genomic diversity. Nature 604, 437–446 (2022).
7.	Rautiainen, M. et al. Telomere-to-telomere assembly of diploid chromosomes with Verkko. Nat. Biotechnol. (2023) doi:10.1038/s41587-023-01662-6.
8.	Butchbach, M. E. R. Copy Number Variations in the Survival Motor Neuron Genes: Implications for Spinal Muscular Atrophy and Other Neurodegenerative Diseases. Front Mol Biosci 3, 7 (2016).
9.	Gao, Y. et al. A pangenome reference of 36 Chinese populations. Nature 619, 112–121 (2023).
10.	Uddin, M. et al. A draft Arab pangenome reference. Research Square (2023) doi:10.21203/rs.3.rs-3490341/v1.
11.	Kulmanov, M. et al. A personal, reference quality, fully annotated genome from a Saudi individual. bioRxiv 2022.11.05.515129 (2022) doi:10.1101/2022.11.05.515129.
12.	Liao, W.-W. et al. A draft human pangenome reference. Nature 617, 312–324 (2023).
13.	Hickey, G. et al. Pangenome graph construction from genome alignments with Minigraph-Cactus. Nat. Biotechnol. (2023) doi:10.1038/s41587-023-01793-w.
14.	Behera, S. et al. Comprehensive and accurate genome analysis at scale using DRAGEN accelerated algorithms. bioRxiv (2024) doi:10.1101/2024.01.02.573821.
15.	Computational Pan-Genomics Consortium. Computational pan-genomics: status, promises and challenges. Brief. Bioinform. 19, 118–135 (2018).
16.	Rehm, H. L. et al. GA4GH: International policies and standards for data sharing across genomic research and healthcare. Cell Genom 1, (2021).
17.	Global Indigenous Data Alliance. https://www.gida-global.org/.
18.	Joly, Y. et al. Establishing the International Genetic Discrimination Observatory. Nat. Genet. 52, 466–468 (2020).
19.	Joly, Y. et al. The Genetic Discrimination Observatory: confronting novel issues in genetic discrimination. Trends Genet. 37, 951–954 (2021).
20.	de Vries, J. et al. The H3Africa policy framework: negotiating fairness in genomics. Trends Genet. 31, 117–119 (2015).
21.	H3AFRICA REPORT ON INFORMED CONSENT AND COMMERCIALISATION. https://h3africa.org/wp-content/uploads/2021/02/H3A_Report-Review-Informed-Consent_FINAL_111120.pdf.
22.	Harnessing Genomic Technologies Toward Improving Health in Africa: Recommendations for the Human Heredity and Health in Africa (H3Africa) Initiative. H3Africa Whitepaper https://h3africa.org/wp-content/uploads/2018/05/h3africa_whitepaper.pdf.
23.	NCIG Governance Framework. https://ncig.anu.edu.au/files/NCIG-Governance-Framework.pdf.
24.	Website. ‘United Nations Declaration on the Rights of Indigenous Peoples.’ n.d. https://www.ohchr.org/sites/default/files/Documents/Publications/Declaration_indigenous_en.pdf.
25.	Tri-Council Policy Statement: Ethical Conduct for Research Involving Humans. https://ethics.gc.ca/eng/policy-politique_tcps2-eptc2_2022.html.
26.	Te Mata Ira Genome Research Guidelines. https://www.waikato.ac.nz/__data/assets/pdf_file/0018/321534/Te-Mata-Ira-Genome-Research-Guidelines.pdf.
27.	Beaton, A. et al. Engaging Māori in biobanking and genomic research: a model for biobanks to guide culturally informed governance, operational, and community engagement activities. Genet. Med. 19, 345–351 (2017).
28.	Berghofer, Q. Genomic partnerships: guidelines for genomic research with Aboriginal and Torres Strait Islander peoples of Queensland. Brisbane: QIMR Berghofer Medical Research Institute (2019).
29.	Tsosie, K. S., Yracheta, J. M., Kolopenuk, J. & Smith, R. W. A. Indigenous data sovereignties and data sharing in biological anthropology. Am. J. Phys. Anthropol. 174, 183–186 (2021).
30.	Tsosie, K. S., Yracheta, J. M. & Dickenson, D. Overvaluing individual consent ignores risks to tribal participants. Nat. Rev. Genet. 20, 497–498 (2019).
31.	American Indian & Alaska Native Genetics Resource Center. American Indian and Alaska Native Genetics Research Resource Guide: Tools for Tribal Leaders and Citizens https://web.archive.org/web/20160327172526/http://genetics.ncai.org/files/NCAI%20genetics%20research%20resource%20guide%20FINAL%20PDF.pdf.
32.	Around Him, D. et al. Tribal IRBs: A Framework for Understanding Research Oversight in American Indian and Alaska Native Communities. Am. Indian Alsk. Native Ment. Health Res. 26, 71–95 (2019).
33.	Carroll, S. R. et al. Using Indigenous Standards to Implement the CARE Principles: Setting Expectations through Tribal Research Codes. Front. Genet. 13, 823309 (2022).
34.	Carroll, S. R. et al. Extending the CARE Principles from tribal research policies to benefit sharing in genomic research. Front. Genet. 13, 1052620 (2022).
35.	NIH facilitates first Tribal data-sharing agreement with Navajo Nation. National Institutes of Health https://www.nih.gov/news-events/news-releases/nih-facilitates-first-tribal-data-sharing-agreement-navajo-nation.
36.	Tauali i, M. et al. Native Hawaiian views on biobanking. J. Cancer Educ. 29, 570–576 (2014).
37.	Hudson, M. Te Ara Tika: Guidelines for Maori Research Ethics: A Framework for Researchers and Ethics Committee Members. (Health Research Council of New Zealand, 2010).
38.	Health and Social Services Committee of the Navajo Nation Council. Approving a Moratorium on Genetic Research Studies Conducted within the Jurisdiction of the Navajo Nation until such time that a Navajo Nation Human Research Code has been Amended by the Navajo Nation Council. Preprint at (2002).
39.	Association of Hawaiian Civic Clubs. Resolution urging the AHCC to support education in genetics and the ethical issues surrounding genetic studies that impact on Native Hawaiians. Preprint at (Adopted November 13, Editor. 2004).
40.	Carroll, S. R. et al. The CARE principles for indigenous data governance. Data Sci. J. 19, (2020).
41.	Carroll, S. R., Herczog, E., Hudson, M., Russell, K. & Stall, S. Operationalizing the CARE and FAIR Principles for Indigenous data futures. Sci Data 8, 108 (2021).
42.	Wilkinson, M. D. et al. The FAIR Guiding Principles for scientific data management and stewardship. Sci Data 3, 160018 (2016).
43.	Lin, D. et al. The TRUST Principles for digital repositories. Sci Data 7, 144 (2020).
44.	Porubsky, D. & Eichler, E. E. A 25-year odyssey of genomic technology advances and structural variant discovery. Cell (2024) doi:10.1016/j.cell.2024.01.002.
45.	Joly, Y., Dove, E., Knoppers, B. M. & Nicol, D. The GA4GH Regulatory and Ethics Work Stream (REWS) at 10: An Interdisciplinary, Participative Approach to International Policy Development in Genomics. in The Law and Ethics of Data Sharing in Health Sciences (eds. Corrales Compagnucci, M., Minssen, T., Fenwick, M., Aboy, M. & Liddell, K.) 13–32 (Springer Nature Singapore, 2024).
46. Vollger, M.R. et al. Increased mutation and gene conversion within human segmental duplications. Nature 617, 325–334 (2023). https://doi.org/10.1038/s41586-023-05895-y
47. Guarracino, A. et al. Recombination between heterologous human acrocentric chromosomes. Nature 617, 335–343 (2023). https://doi.org/10.1038/s41586-023-05976-y
